esmvaltool/cmorizers/obs/cmorize_obs_cds_satellite_albedo.py

```diff
@@ -0,0 +1,218 @@
+"""ESMValTool CMORizer for cds-satellite-albedo data.
+
+Tier
+   Tier 3
+
+Source
+   https://cds.climate.copernicus.eu/
+     cdsapp#!/dataset/satellite-albedo?tab=form
+
+Last access
+   20190401
+
+Download and processing instructions
+   - Download the data from source to the right directory
+     using the download script
+   - Decompress the files within the directory:
+       "find . -name '*.tar.gz' -execdir tar -xzvf '{}' \;"
```

Comment:
```suggestion
       find . -name '*.tar.gz' -execdir tar -xzvf '{}'
```

Reply:
test is failing here @bascrezee :beer:

Reply:
Yes, I noticed too, but I want that instruction in. The suggestion you did does not extract the files. Trying out if this works now: https://stackoverflow.com/questions/5033727/how-do-i-get-pyflakes-to-ignore-a-statement

Reply:
This works now. :)  # NOQA